packages/universal-xml/package.json

```diff
@@ -27,7 +27,7 @@
     "@appium/base-plugin": "^1.7.0",
     "appium-base-driver": "^8.0.0-beta.4",
     "fast-xml-parser": "^3.17.6",
-    "xmldom": "^0.4.0",
+    "xmldom": "^0.6.0",
```

Comment:
I'd rather simply change it to `^0.x`, also for others with major version number 0